blog: Guide to Speaking at PyCon India (#413)
pythonindia/inpycon-blog: content/blog/2023/2023_guide_to_speaking.md

Title: Guide to Speaking at PyCon India
Category: 2023

> Slug: guide-to-speaking

Reviewer: Please add all the required metadata, you can see in the README what all is required.

**Hello Hello Pythonistas!!!**

*Are you all ready to witness the largest gathering of Pythonistas in India?*

> The largest gathering of Pythonistas in India, [**PyCon India**](https://in.pycon.org/2023/) is back. The 14th PyCon India will be held online 29 Sept - 02 Oct in Hyderabad. So, whether you're someone who is submitting a presentation idea for the first time or you want to share what you have been working on with Python for a while or just want to share your tales with Python, this is the place for you. Your moment to shine has come!

Reviewer: We'll not be online. We'll be  back offline after 4 years.

We're excited to see beginning, intermediate, and advanced proposals on a range of topics from speakers with varying experience and skillsets. You don't have to be a veteran speaker to submit a proposal. We are seeking people of all backgrounds to help our conference be really diverse. 

**Why should you speak at PyCon India?**

1. Get a chance to contribute to the Python community
2. Share your Experiences and Perspectives with a Diverse Audience
3. Receive valuable feedback & network with Python Experts
4. Foster Innovation & Inspire others!

> Now that we have understood why one should submit a CFP, lets quickly dive into the things to keep in mind when drafting your CFP. We will discuss how to submit your CFP and provide some steps for new speakers to take the stage at PyCon India.

Reviewer: It should be let's

> - ***Keep a track of the Deadlines***: Keep an eye on the deadline so that you don't forget on missing to submit the proposal. Start preparing now and avoid the last minute hassel.

Reviewer: ```suggestion
- ***Keep track of the Deadlines***: Keep an eye on the deadline so that you don't forget on missing to submit the proposal. Start preparing now and avoid the last-minute hassle.
```

-  ***Craft an Engaging CFP***: Submitting a compelling CFP is crucial to capture the attention of the PyCon selection committee. Here are some tips to make your CFP stand out.

> 	-  ***Knowing the CFP requirements***: Go to the PyCon India website [link here] and read up on the CFP guidelines & categories. Pay attention to the topic categories, formatting specifications, and submission dates.

Reviewer: https://in.pycon.org/cfp/pycon-india-2023/proposals/

	-  ***Choosing a Topic***: If this is your first time speaking and you're unsure about what to talk about, you can choose any interesting project you've worked on, such as an intriguing Python project you built or your experience using Python to solve a difficult problem.

> 	-  ***Choose a captivating title***: Your title is the first thing that will get noticed by the PyCon committe and the audience. It's your elevator pitch hence make sure to create a concise and attention-grabbing title that accurately represents the content of your talk.

Reviewer: ```suggestion
	-  ***Choose a captivating title***: Your title is the first thing that will get noticed by the PyCon India committee and the audience. It's your elevator pitch hence make sure to create a concise and attention-grabbing title that accurately represents the content of your talk.
```

	-  ***Define your target audience***: Clearly define your talk's intended audience, whether it's novices, intermediate developers, or domain specialists.

	-  ***Outline the talk's objectives***: Clearly outline what participants can expect from your session and what they will learn from attending your talk. Describe the issue you intend to address and the potential solutions or insights you intend to provide.

	-  ***Provide an outline or structure***: Give a brief overview of the talk's structure, including the key points or sections you will cover. This helps in demonstrating a well-organized presentation.

	-  ***Be authentic***: Share real life practical examples, use cases, or projects that demonstrate the application of the concepts you plan to discuss. This helps the reviewers see the tangible benefits of your talk. Try to include personal experiences in order to connect with the audience.

	-  ***Iterate and seek feedback***: Before submitting your CFP, seek feedback from your colleagues or friends. Incorporate their suggestions and iterate on your proposal to make it stronger.

>   Here is a list of the past talks for you to refer back to [insert link]

Reviewer: You can find some of the previous conferences on https://in.pycon.org/cfp

4. Foster Innovation & Inspire others!

Now that we have understood why one should submit a CFP, let's quickly dive into the things to keep in mind when drafting your CFP. We will discuss how to submit your CFP and provide some steps for new speakers to take the stage at PyCon India.

- ***Keep track of the Deadlines***: Keep an eye on the deadline so that you don't forget on missing to submit the proposal. Start preparing now and avoid the last-minute hassle.

-  ***Craft an Engaging CFP***: Submitting a compelling CFP is crucial to capture the attention of the PyCon selection committee. Here are some tips to make your CFP stand out.

	-  ***Knowing the CFP requirements***: Go to the [PyCon India website](https://in.pycon.org/cfp/pycon-india-2023/proposals/) and read up on the CFP guidelines & categories. Pay attention to the topic categories, formatting specifications, and submission dates.

	-  ***Choosing a Topic***: If this is your first time speaking and you're unsure about what to talk about, you can choose any interesting project you've worked on, such as an intriguing Python project you built or your experience using Python to solve a difficult problem.

> 	-  ***Choose a captivating title***: Your title is the first thing that will get noticed by the PyCon India committe and the audience. It's your elevator pitch hence make sure to create a concise and attention-grabbing title that accurately represents the content of your talk.

Reviewer: ```suggestion
	-  ***Choose a captivating title***: Your title is the first thing that will get noticed by the PyCon India committee and the audience. It's your elevator pitch hence make sure to create a concise and attention-grabbing title that accurately represents the content of your talk.
```

	-  ***Define your target audience***: Clearly define your talk's intended audience, whether it's novices, intermediate developers, or domain specialists.

	-  ***Outline the talk's objectives***: Clearly outline what participants can expect from your session and what they will learn from attending your talk. Describe the issue you intend to address and the potential solutions or insights you intend to provide.

	-  ***Provide an outline or structure***: Give a brief overview of the talk's structure, including the key points or sections you will cover. This helps in demonstrating a well-organized presentation.

	-  ***Be authentic***: Share real life practical examples, use cases, or projects that demonstrate the application of the concepts you plan to discuss. This helps the reviewers see the tangible benefits of your talk. Try to include personal experiences in order to connect with the audience.

	-  ***Iterate and seek feedback***: Before submitting your CFP, seek feedback from your colleagues or friends. Incorporate their suggestions and iterate on your proposal to make it stronger.

> 	You can find some of the previous conferences on https://in.pycon.org/cfp

Reviewer: ```suggestion
	You can find some of the previous conferences at https://in.pycon.org/cfp
```

	-  ***Define your target audience***: Clearly define your talk's intended audience, whether it's novices, intermediate developers, or domain specialists.

	-  ***Outline the talk's objectives***: Clearly outline what participants can expect from your session and what they will learn from attending your talk. Describe the issue you intend to address and the potential solutions or insights you intend to provide.

	-  ***Provide an outline or structure***: Give a brief overview of the talk's structure, including the key points or sections you will cover. This helps in demonstrating a well-organized presentation.

	-  ***Be authentic***: Share real life practical examples, use cases, or projects that demonstrate the application of the concepts you plan to discuss. This helps the reviewers see the tangible benefits of your talk. Try to include personal experiences in order to connect with the audience.

	-  ***Iterate and seek feedback***: Before submitting your CFP, seek feedback from your colleagues or friends. Incorporate their suggestions and iterate on your proposal to make it stronger.

> 	You can find some of the previous conferences at https://in.pycon.org/cfp

Reviewer: ```suggestion
	You can find some of the previous conferences at <https://in.pycon.org/cfp>.
```
